### ISSUE TITLE
[Feature]: Changing the flag to US flag in currency widget's default option

### ISSUE THREAD

**Issue by Rishabhkaul** [appsmithorg/appsmith#22167]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

The default option when using the currency widget is a lesser known flag corresponding to USD. It should probably be US flag.  

<img width="291" alt="Screenshot 2023-04-06 at 17 04 02" src="https://user-images.githubusercontent.com/1650391/230435178-6731a68b-9bb0-4b0f-b0dc-33e6690cff35.png">

### Why should this be worked on?

US $ is a reasonable default to have in the currency input. However it makes sense to put the US flag there, instead of the current option (I don't know what country that is) since for most cases where people will use US $, they're probably based in USA. 

I'd also imagine it might be a relatively minor fix.

**Comment by Nikhil-Nandagopal** [comment]
@Rishabhkaul congrats on opening your first issue :) 
@dilippitchika this should be a pretty minor fix right?

**Comment by somangshu** [comment]
We have multiple conflict around this, specially the US - types (even comes up with country codes), here is a similar issue #18136

@dilippitchika did document the way forward [here](https://github.com/appsmithorg/appsmith/issues/18136#issuecomment-1397928885)

**Comment by somangshu** [comment]
@dilippitchika can you frame what the final option needs to look like, we will modify our object accordingly!

**Comment by dilippitchika** [comment]
@somangshu https://www.loom.com/share/a50c531a48fb475bab890d7b500148c4

**Comment by somangshu** [comment]
@dilippitchika makes sense, so for the currency thing we will do `currency symbol - currency code` > `₹ - INR`

While we are thinking about this, Should we also rectify the problem with country code in phone input widget? #18136

**Comment by dilippitchika** [comment]
We can try it out in phone input but there we need to write the countries because +1 or +2 on it's own is not clear to users. So we have to mention all countries either in short code or full name in the field. But that's a ux issue.

**Comment by Vishrut19** [comment]
Hey! Is anyone working on this issue? Or is anyone open to collaborating?

**Comment by somangshu** [comment]
Thanks for showing interest on the issue @Vishrut19, We would like to manage this issue internally. 
Can you please go though other `good first issue` and see is something is interesting to you
